packages/devtools-launchpad/bin/firefox-proxy

```diff
@@ -6,8 +6,8 @@ const ws = require("ws");
 const net = require("net");
 
 function proxy(webSocketPort, tcpPort, logging) {
-  console.log("Listening for WS on *:" + webSocketPort);
-  console.log("Will proxy to TCP on *:" + tcpPort + " on first WS connection");
+  console.log("Listening for WS on *:" + webSocketPort + ". " +
```

Comment:
Hmm. @adymitruk I'd be okay remove this line entirely. 

This is now very advanced usage